snap/snapcraft.yaml

```diff
@@ -68,4 +68,4 @@ parts:
       - libxdmcp6
       - libglu1-mesa
     build-attributes:
-      - no-patchelf
+      - no-patchelf
```

Comment:
Can you add a newline on the end of this file please?

Reply:
Sure.. doing all the formatting